Podcast/SearchTableViewController.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import UIKit
+import NVActivityIndicatorView
```

Comment:
is this the activity indicator view we're going to be using everywhere? if so I think we should make an extension on UIViewController to show the indicator view (kind of like I did with the action sheet) so you don't have to import and configure the activity indicator in every VC